axolotl-web/src/components/Message.vue

```diff
@@ -241,6 +241,18 @@ video,
     max-width: 100%;
     max-height: 80vh;
 }
+.outgoing .attachment-img {
+  background: center center no-repeat;
+  background-color: #000;
+  background-image: url("../assets/images/loading.svg");
+
+  img {
+    opacity: 0.2;
+  }
+}
+.sent .attachment-img img {
```

Comment:
Please confirm that `message.IsSent` is kept even after read and delivered. That's what I observed but I want a confirmation.

Reply:
Is Sent isn't changed after sending a message so you can rely on it.